posts/2024-02-13-24.0.0.2-beta.adoc

```diff
@@ -0,0 +1,173 @@
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dmuelle
+author_github: https://github.com/dmuelle
+seo-title: Back-channel logout support for OpenID Connect clients and servers in 24.0.0.2-beta - OpenLiberty.io
+seo-description: OpenID Connect clients and servers in Open Liberty now support back-channel logout. Back-channel logout allows OpenID Connect servers to directly notify OpenID Connect clients of a user logout so each OpenID Connect client can also log the user out locally.
+blog_description: OpenID Connect clients and servers in Open Liberty now support back-channel logout. Back-channel logout allows OpenID Connect servers to directly notify OpenID Connect clients of a user logout so each OpenID Connect client can also log the user out locally.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Back-channel logout support for OpenID Connect clients and servers in 24.0.0.2-beta
+David Mueller <https://github.com/dmuelle>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post
+
+OpenID Connect clients and servers in Open Liberty now support back-channel logout. Back-channel logout allows OpenID Connect servers to directly notify OpenID Connect clients of a user logout so each OpenID Connect client can also log the user out locally.
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27362
+// Contact/Reviewer: jimmy1wu
+// // // // // // // //
+
+Previously, OpenID Connect servers could notify OpenID Connect clients that a user logged out only by using iframes that were embedded in the OpenID Connect client's web page. If the web page wasn't active, the OpenID Connect client wasn't notified of the logout that occurred on the OpenID Connect server. Back-channel logout solves this problem through direct communication between the OpenID Connect server and clients.
+
+== Back-channel logout for OpenID Connect clients
+
+With this release, back-channel logout for OpenID clients is enabled by default when the link:{url-prefix}/docs/latest/reference/feature/openidConnectClient.html[OpenID Connect Client] and link:{url-prefix}/docs/latest/reference/feature/socialLogin.html[Social Media Login] features are enabled. The following back-channel logout endpoints are enabled for those features:
+
+* OpenID Connect Client: `/oidcclient/backchannel_logout/{clientId}`
+* Social Media Login: `/ibm/api/social-login/backchannel_logout/{clientId}`
+
+== Back-channel logout for OpenID Connect servers
+
+Two options are available to enable back-channel logout for OpenID Connect servers:
+
+* Use the `backchannel_logout_uri` metadata value to specify the OpenID Connect client's back-channel logout URI when the OAuth client is dynamically registered.
```

Comment:
we dont really say how to do this- can we provide more context or a link for more info

Reply:
```
* Use the `backchannel_logout_uri` metadata value to specify the OpenID Connect client's back-channel logout URI when the OAuth client is dynamically registered using the link:https://www.ibm.com/docs/en/was-liberty/base?topic=cocpil-configuring-openid-connect-provider-accept-client-registration-requests[OpenID Connect provider's client registration endpoint].
```

Reply:
updated